### DISCUSSION TITLE
Can it be trained for style ?

### DISCUSSION THREAD

**Discussion by scarbain** [lllyasviel/ControlNet#26] [General]
Let's say we give an image style (for example a painting of Van Gogh) as control image and the target must match that style but following the prompt.
Could it work ? Or does the informations the control images carry are too much based on the position of the pixels inside ? (because all the given examples from the repo are based on positional information)

**Comment by geroldmeisinger** [comment]
[Composer: Creative and Controllable Image Synthesis with Composable Conditions](https://arxiv.org/abs/2302.09778)
